tests/scripts/test_scripts.py

```diff
@@ -232,93 +225,84 @@ def test_train_dagger_main(tmpdir):
     assert isinstance(run.result, dict)
 
 
-def test_train_dagger_warmstart(tmpdir):
+def test_train_bc_main(tmpdir):
     run = train_imitation.train_imitation_ex.run(
-        command_name="dagger",
-        named_configs=["cartpole"] + ALGO_FAST_CONFIGS["imitation"],
-        config_updates=dict(
-            common=dict(log_root=tmpdir),
-            demonstrations=dict(rollout_path=CARTPOLE_TEST_ROLLOUT_PATH),
-            dagger=dict(
-                expert_policy_type="ppo",
-                expert_policy_path=CARTPOLE_TEST_POLICY_PATH,
-            ),
-        ),
+        command_name="bc",
+        named_configs=["seals_cartpole"] + ALGO_FAST_CONFIGS["imitation"],
+        config_updates=dict(common=dict(log_root=tmpdir)),
     )
     assert run.status == "COMPLETED"
-
-    log_dir = pathlib.Path(run.config["common"]["log_dir"])
-    policy_path = log_dir / "scratch" / "policy-latest.pt"
-    run_warmstart = train_imitation.train_imitation_ex.run(
-        command_name="dagger",
-        named_configs=["cartpole"] + ALGO_FAST_CONFIGS["imitation"],
-        config_updates=dict(
-            common=dict(log_root=tmpdir),
-            demonstrations=dict(rollout_path=CARTPOLE_TEST_ROLLOUT_PATH),
-            dagger=dict(
-                expert_policy_type="ppo",
-                expert_policy_path=CARTPOLE_TEST_POLICY_PATH,
-            ),
-            agent_path=policy_path,
-        ),
-    )
-    assert run_warmstart.status == "COMPLETED"
-    assert isinstance(run_warmstart.result, dict)
+    assert isinstance(run.result, dict)
 
 
-def test_train_dagger_error_and_exceptions(tmpdir):
-    with pytest.raises(Exception, match=".*expert_policy_path cannot be None.*"):
+def test_train_bc_main_with_none_demonstrations_raises_value_error(tmpdir):
+    with pytest.raises(ValueError, match=".*n_expert_demos.*rollout_path.*"):
         train_imitation.train_imitation_ex.run(
-            command_name="dagger",
-            named_configs=["cartpole"] + ALGO_FAST_CONFIGS["imitation"],
+            command_name="bc",
+            named_configs=["seals_cartpole"] + ALGO_FAST_CONFIGS["imitation"],
             config_updates=dict(
                 common=dict(log_root=tmpdir),
-                demonstrations=dict(rollout_path=CARTPOLE_TEST_ROLLOUT_PATH),
-                dagger=dict(
-                    expert_policy_type="ppo",
-                    expert_policy_path=None,
-                ),
+                demonstrations=dict(n_expert_demos=None),
             ),
         )
 
 
-def test_train_bc_main(tmpdir):
+def test_train_bc_main_with_expert_path(tmpdir):
     run = train_imitation.train_imitation_ex.run(
         command_name="bc",
-        named_configs=["cartpole"] + ALGO_FAST_CONFIGS["imitation"],
+        named_configs=["seals_cartpole"] + ALGO_FAST_CONFIGS["imitation"],
         config_updates=dict(
             common=dict(log_root=tmpdir),
-            demonstrations=dict(rollout_path=CARTPOLE_TEST_ROLLOUT_PATH),
```

Comment:
I'm OK with the test as-written but I do want to flag that loading rollouts directly is the *most realistic* use case as it's the only option for human demos (even if for quick experiments we more often use synthetic data). So, I don't want rollouts to be treated as a second-class citizen. Ideally we have tests for both (could parametrize fairly easily, or just mix-and-match).

Reply:
I added that case back again.